src/pages/resultsView/cancerSummary/CancerSummaryChart.tsx

```diff
@@ -105,7 +105,7 @@ export class CancerSummaryChart extends React.Component<CancerSummaryChartProps,
     }
 
     private get bottomPadding(){
-        return this.longestXLabelLength! * 7 + 40;
+        return this.longestXLabelLength! * 5 + 40;
```

Comment:
you could try to do the explicit calculation... getTextWidth * Math.cos (or is it sin) of the rotation angle. may not be worth it if this is good enough. thats how I did it for plots tab though, just want to lyk